cli/command/formatter/stats.go

```diff
@@ -145,8 +145,10 @@ func (c *containerStatsContext) Container() string {
 
 func (c *containerStatsContext) Name() string {
 	c.AddHeader(nameHeader)
-	name := c.s.Name[1:]
-	return name
+	if len(c.s.Name) > 1 {
+		return c.s.Name[1:]
```

Comment:
We should eliminate those `/` at all. They have no real value now. I'm ok with slicing.